ENIGMAsystem/SHELL/Universal_System/Makefile

```diff
@@ -1,3 +1,6 @@
-SOURCES += $(wildcard Universal_System/*.cpp) $(wildcard Universal_System/Object_Tiers/*.cpp) $(wildcard Universal_System/Instances/*.cpp)
+SOURCES += $(wildcard Universal_System/*.cpp)\
+           $(wildcard Universal_System/Object_Tiers/*.cpp)\
+           $(wildcard Universal_System/Instances/*.cpp)\
+           $(wildcard Universal_System/Resources/*.cpp)
```

Comment:
@JoshDreamland is it ok to align a makefile variable using spaces instead of tabs? Do tabs only apply to rules?

Reply:
I guess that's fine, as long as you have the tabs from the previous line (in this case, there aren't any).

Reply:
Ok, good. We haven't hit any snags with it, so we'll go with it until someone cries.

Reply:
Alternatively, I could have formatted it like the following. Still can if and when somebody does cry.
```make
SOURCES +=
	$(wildcard Universal_System/*.cpp)\
	$(wildcard Universal_System/Object_Tiers/*.cpp)\
	$(wildcard Universal_System/Instances/*.cpp)\
	$(wildcard Universal_System/Resources/*.cpp)
```